Assignment2-Python/deletemiddlenode.py

```diff
@@ -0,0 +1,51 @@
+#2.3 Delete Middle Node (p. 93)
+class Node:
+    def __init__(self, data=0, next=None):
+        self.data = data
+        self.next = next
+
```

Comment:
The spacing convention in Python is to separate top-level class definitions from top-level function definitions using two new lines (instead of one).